CONTRIBUTING.md

```diff
@@ -268,6 +270,306 @@ The generators are covered by generator tests using Rails's generator testing he
 
 `rake run_rspec:shakapacker_examples_basic` is a great way to run tests on one generator. Once that works, you should run `rake run_rspec:shakapacker_examples`. Be aware that this will create a huge number of files under a `/gen-examples` directory. You should be sure to exclude this directory from your IDE and delete it once your testing is done.
 
+#### Manual Generator Testing Workflow
+
+For comprehensive testing of generator changes, use this manual testing workflow with dedicated test applications:
+
+**1. Set up test application with clean baseline:**
+
+```bash
+# Navigate to test app directory (replace with your test app)
+cd ~/shakacode/react-on-rails/react_on_rails-test-apps/react-on-rails-tutorial-v15
```

Comment:
Need to use something like

`{project_dir}/react_on_rails` (this repo)
`{project_dir}/test-app` (where you will test the generator)